demo/demo-app.jsx

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import { Form, Field } from 'react-final-form';
 import { Form as PfForm, Col, Row, Button, Grid } from 'patternfly-react';
 import { FinalFormSwitch } from '../src/forms';
+import { translate, sprintf } from '../src/global-functions';
```

Comment:
This should read `import { __, sprintf } ...`, right?

Reply:
@mzazrivec oh this file should not even be in this PR. Thanks